### PR TITLE
bug(tv): reduction was handled to return proper values

### LINKED ISSUE #65: Same values for `mean` and `sum` reductions of TV loss.
**Describe the bug**
Reductions at `TVLoss` return the same value, when default parameters are used to initialise `TVLoss()`

**To Reproduce**
```python
from photosynthesis_metrics import TVLoss
import torch
torch.manual_seed(42)
a = torch.rand(3,3,256,256)
print(f'mean reduction: {TVLoss(reduction='mean')(a)}')
print(f'sum reduction: {TVLoss(reduction='sum')(a)}')
```
Output
```python
>>> mean reduction: 255.3401
>>> sum reduction: 255.3401
```

**Expected behavior**
Output
```python
>>> mean reduction: 255.3401
>>> sum reduction: 766.0204
```

**Solution**
- Get rid of `size_average` parameter at `TVLoss`;
- Call `total_variation` with `size_average=False` at `TVLoss().forward()`

### PR DESCRIPTION
**Pull request by denproc**
Closes #65 issue. Related to #64.

**Proposed changes:**
- `size_average` parameter was eliminated from `TVLoss` initialisation;
- `total_variation` is called with `size_average=False`;
- `reduction` is checked to be `"none"|"mean"|"sum"`.

### CODE REVIEW

**Comment by denproc** [comment]
If we would like to postpone the bug fix, we can do it here, but after merging #55.

**Comment by zakajd** [comment]
> If we would like to postpone the bug fix, we can do it here, but after merging #55.

Yes, let's merge #55  first.

**Comment by denproc** [comment]
Ready for review. Now it also resolves #64

File: photosynthesis_metrics/tv.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def _adjust_tensor_dimensions(x: torch.Tensor):
 def _validate_input(x: torch.Tensor) -> None:
     """Validates input tensor"""
     assert isinstance(x, torch.Tensor), f'Input must be a torch.Tensor, got {type(x)}.'
-    assert 1 < x.dim() < 5, f'Input image must be 4D tensor, got image of shape {x.size()}.'
```
Comment (snk4tr) [inline photosynthesis_metrics/tv.py:26]: Good catch!
